### ISSUE TITLE
Switching from English to Spanish

### ISSUE THREAD

**Issue by ExcellaCenter** [ak1394/react-native-tts#9]
Using AVD which is emulating android 6.0, API 32, Intel Atom (x86_64)
"react": "15.4.2",
"react-native": "^0.38.0",
"react-native-tts": "0.4.0"

Steps to reproduce
Tts.setDefaultLanguage('en-US')
Tts.speak('text')
Tts.setDefaultLanguage('es-ES)

I get the alert: Unfortunately, Pico TTS has stopped.  After the initial alert, the spanish version will play.  However, if I switch back and forth a couple of times, it stops working and causes this error in the debugger:
Possible Unhandled Promise Rejection (id: 0):unable to play
Error: unable to play
    at createErrorFromErrorData (http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22591:11)
    at http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22554:35
    at MessageQueue.__invokeCallback (http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22940:10)
    at http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22792:8
    at guard (http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22711:1)
    at MessageQueue.invokeCallbackAndReturnFlushedQueue (http://localhost:8081/index.android.bundle?platform=android&dev=true&hot=true&minify=false:22791:1)
    at http://localhost:8081/debuggerWorker.js:71:58

**Comment by ExcellaCenter** [comment]
Update, this seems to be specific to the Intel Atom (x86_64) image.  Moving to the Atom (x86) image solves the issue and I am not getting the crash happening on my handset (Nexus 6p).  However, any insight would be appreciated.

**Comment by ak1394** [comment]
Perhaps there is the issue with second setDefaultLanguage() call being executed at the same time with the first one? If you want to perform changes one after the another, you should do something along the lines of:

Tts.setDefaultLanguage('en-US').then(function(result) {
 // this function will be called after the first setDefaultLanguage() has been executed
  Tts.speak('text').then(funciton (result) {
  // now let's try to set the second language 
  Tts.setDefaultLanguage('es-ES);
  });
})

**Comment by ExcellaCenter** [comment]
I was actually considering the same thing.  I thought they might be stepping on each other.  So I rewrote it as follows:

Tts.stop()
      .then(()=>Tts.setDefaultLanguage(language))
      .then(()=>Tts.speak(phrase))

My thinking being that it would only change the language after clearing the queue.  Then only speak after the language was finished being set.  I was really hopeful but it didn't seem to make a difference.

I also realized that I didn't mention that I am running Linux Mint.  I am curious if anyone else is getting this same issue in the Intel Atom (x86_64) configuration.
